src/core/expr/fexpr_cumsumprod.cc

```diff
@@ -69,9 +71,12 @@ class FExpr_cumsum : public FExpr_Func {
     Column evaluate1(Column&& col, const Groupby& gby) const {
       SType stype = col.stype();
       switch (stype) {
-        case SType::VOID:    return Column(new ConstInt_ColumnImpl(
+        case SType::VOID:    if (MIN) {return Column(new ConstInt_ColumnImpl(
                                       col.nrows(), 0, SType::INT64
-                                    ));
+                                    ));}
+                             else {return Column(new ConstInt_ColumnImpl(
+                                      col.nrows(), 1, SType::INT64
+                                    ));}
```

Comment:
No need for brackets when there is just one line of code. Removing them will improve readability.

Reply:
The brackets around the return statement. If you want to keep them, it is better to move the return to the next line. Or simply remove them.